MSProject/ViewControllers/LibraryTableView/ViewModel/WBLibraryViewModel.swift

```diff
@@ -18,50 +20,35 @@ enum SortMethod {
 
 class WBLibraryViewModel {
     
-    private var libraryItems: [WBBook] = []
-
-    private var bookViewModels: [WBBookViewModel] = [] {
-        didSet {
-            reloadViewClosure?()
-        }
+    var libraryItems: [WBBook] = []
+//    private let libraryItems = MutableProperty<[WBBook]>([])
```

Comment:
this seems like a good idea

Reply:
Yeah, is there any reason why you didn't end up implementing it?

Reply:
cause i didnt understand how to use it... :p